tests/test_image_resize.py

```diff
@@ -154,7 +154,7 @@ def test_resize_image_change_to_gif(self):
 
         image = Image.open(BytesIO(image_data)).resize(
             (50, 50),
-            Image.ANTIALIAS
+            Image.Resampling.LANCZOS
```

Comment:
`Image.ANTIALIAS` was removed in the 10.0.0 released https://pillow.readthedocs.io/en/stable/releasenotes/10.0.0.html#constants